ImputationPipeline/CreateAggregationSets/CreateAggregationSets.py

```diff
@@ -0,0 +1,235 @@
+                raise RuntimeError(f'ERROR: {upload_response.text}')
+            # Add date and time created to sample_set
+            print(f'Adding date and time to newly created {table_name}_sets...')
+
+            now = str(datetime.now(pytz.timezone('US/Eastern')))
+            for i, (this_lab_batch, this_aggregation_set) in enumerate(added_sample_sets_dict.items()):
+                update_response = fapi.update_entity(self.workspace_namespace, self.workspace_name,
+                                                     f'{table_name}_set', this_aggregation_set.set_id,
+                                                     [{"op": "AddUpdateAttribute",
+                                                       "attributeName": "time_sample_set_updated",
+                                                       "addUpdateAttribute": now},
+                                                      {"op": "AddUpdateAttribute", "attributeName": "delivered",
+                                                       "addUpdateAttribute": False},
+                                                      {"op": "AddUpdateAttribute", "attributeName": "redeliver",
+                                                       "addUpdateAttribute": False},
+                                                      {"op": "AddUpdateAttribute", "attributeName": "group",
+                                                       "addUpdateAttribute": this_aggregation_set.group},
+                                                      {"op": "AddUpdateAttribute", "attributeName": "lab_batch",
+                                                       "addUpdateAttribute": this_aggregation_set.lab_batch}
+                                                      ])
+                if not update_response.ok:
+                    raise RuntimeError(f'ERROR: {update_response.text}')
+                print(f'    Completed {i + 1}/{len(added_samples_dict)}')
+
+            print(f'Updating rework field in {table_name} table')
+            for samples_list in added_samples_dict.values():
+                for sample in samples_list:
+                    update_response = fapi.update_entity(self.workspace_namespace, self.workspace_name,
+                                                         f'{table_name}', sample,
+                                                         [{"op": "AddUpdateAttribute", "attributeName": "rework",
+                                                           "addUpdateAttribute": False}])
+                    if not update_response.ok:
+                        raise RuntimeError(f'ERROR: {update_response.text}')
+            # Uploading new sample_set table
+            print('SUCCESS')
+            print(f'Printing update {table_name}_set_membership.tsv:')
+            os.system(f'cat CreateSampleSets_data/new_{table_name}_set_membership.tsv')
+        os.system('rm -r CreateSampleSets_data')
+
+
+def main(workspace_namespace, workspace_name):
```

Comment:
Very unimportant, but you could rename this so it looks nicer when you execute it in a notebook for example. Like `run`, `process` or `create_aggregation_sets`?